pluggy.py

```diff
@@ -581,7 +595,7 @@ def varnames(func):
         try:
             func = func.__init__
         except AttributeError:
-            return ()
+            return (), ()
```

Comment:
as an example, if we had used this in pytest and didn't do vendoring, this would be a release that breaks pytest